hack/update-codegen.sh

```diff
@@ -25,7 +25,7 @@ SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 ROOT_PKG=github.com/kubeflow/mxnet-operator
 
 # Grab code-generator version from go.sum
-CODEGEN_VERSION=$(grep 'k8s.io/code-generator' go.sum | awk '{print $2}' | head -1)
+CODEGEN_VERSION=$(grep 'k8s.io/code-generator' go.sum | awk '{print $2}' | sed 's/go.mod//g' | head -1)
```

Comment:
` echo 'k8s.io/code-generator v0.15.10/go.mod h1:G8bQwmHm2eafm5bgtX67XDZQ8CWKSGu9DekI+yN4Y5I=' | awk '{print $2}' | head -1 | sed 's/go.mod//g'`

The result is `v0.15.10/`

I think we need to change to `sed 's/\/go.mod//g'` which has a extra `/`

Reply:
Good catch. Done. Although it would work even with extra /